### DISCUSSION TITLE
date and time functions

### DISCUSSION THREAD

**Discussion by JeanMichelCot** [pawelsalawa/sqlitestudio#3929] [Q&A]
SQLiteStudio 3.2.1 on Windows 10

In the editor, I tried
select datetime(strftime('%s','now'));
result NULL
Did this means the time value (in seconds) given by strftime() can't be the input time value for the datetime function ?

Thank's for help !

**Comment by pawelsalawa** [comment]
According to https://sqlite.org/lang_datefunc.html :
```
datetime(time-value, modifier, modifier, ...)
```
where the `time-value` is described in point 2 on that page to be any of:
* YYYY-MM-DD
* YYYY-MM-DD HH:MM
* YYYY-MM-DD HH:MM:SS
* YYYY-MM-DD HH:MM:SS.SSS
* YYYY-MM-DDTHH:MM
* YYYY-MM-DDTHH:MM:SS
* YYYY-MM-DDTHH:MM:SS.SSS
* HH:MM
* HH:MM:SS
* HH:MM:SS.SSS
* now
* DDDDDDDDDD

None of above represents number of seconds (aka Unix time) that was returned from `strftime('%s','now')`.
